app/models/video.py

```diff
@@ -0,0 +1,20 @@
+from app import db
+from sqlalchemy.orm import relationship
+
+class Video(db.Model):
+    id = db.Column(db.Integer, primary_key=True, nullable=False)
+    title = db.Column(db.String, nullable=False)
+    release_date = db.Column(db.DateTime())
+    total_inventory = db.Column(db.Integer, default = 0)
+    available_inventory = db.Column(db.Integer)
```

Comment:
Like with `Customer`, we can compute `available_inventory` with a relationship:
```rentals = db.relationship('Rental', back_populates='video', lazy=True)```

Then in `to_dict`,  the `available_inventory` isthe total inventory minus length of the `rentals`.
```available_inventory": self.total_inventory - len(self.rentals)```